platforms/Docker/bullseye/14.5/Dockerfile

```diff
@@ -6,17 +6,19 @@ RUN set -ex \
   && git clone https://github.com/plv8/plv8 \
   && cd plv8 \
   && git checkout r3.1 \
+  && make DOCKER=1 generate_upgrades \
   && make DOCKER=1 install \
   && strip /usr/lib/postgresql/14/lib/plv8-3.1.4.so
 
 
 FROM postgres:14.5-bullseye
 
 COPY --from=builder /usr/lib/postgresql/14/lib/plv8* /usr/lib/postgresql/14/lib/
+COPY --from=builder /usr/lib/postgresql/14/lib/bitcode/plv8-3.1.4.index.bc /usr/lib/postgresql/14/lib/bitcode/
+COPY --from=builder /usr/lib/postgresql/14/lib/bitcode/plv8-3.1.4/* /usr/lib/postgresql/14/lib/bitcode/plv8-3.1.4/
```

Comment:
my goal had been to make this dockerfile as version agnostic as I could for plv8 - is there a specific reason you're changing this from a `*` to a specific version?

Reply:
The copy result below was not what I expected for the directory copy.
(change destination /usr/lib/postgresql/14/**lib**/bitcode/
```
COPY --from=builder /usr/lib/postgresql/14/lib/bitcode/plv8* /usr/lib/postgresql/14/lib/bitcode/
```
but, directories were never copied with wildcards.
```
/usr/lib/postgresql/14/lib/bitcode/plv8-3.1.4.index.bc
/usr/lib/postgresql/14/lib/bitcode/plv8_allocator.bc
/usr/lib/postgresql/14/lib/bitcode/plv8.bc
/usr/lib/postgresql/14/lib/bitcode/plv8_func.bc
/usr/lib/postgresql/14/lib/bitcode/plv8_param.bc
/usr/lib/postgresql/14/lib/bitcode/plv8_type.bc
```

Expected copy result

```
/usr/lib/postgresql/14/lib/bitcode/plv8-3.1.4
/usr/lib/postgresql/14/lib/bitcode/plv8-3.1.4/plv8_type.bc
/usr/lib/postgresql/14/lib/bitcode/plv8-3.1.4/plv8_func.bc
/usr/lib/postgresql/14/lib/bitcode/plv8-3.1.4/livescript.bc
/usr/lib/postgresql/14/lib/bitcode/plv8-3.1.4/plv8_allocator.bc
/usr/lib/postgresql/14/lib/bitcode/plv8-3.1.4/plv8.bc
/usr/lib/postgresql/14/lib/bitcode/plv8-3.1.4/plv8_param.bc
/usr/lib/postgresql/14/lib/bitcode/plv8-3.1.4/coffee-script.bc
/usr/lib/postgresql/14/lib/bitcode/plv8-3.1.4.index.bc
```

Or maybe write the version in ENV? 
```
ENV PLV8_VERSION=3.1.4
COPY --from=builder /usr/lib/postgresql/14/lib/bitcode/plv8-${PLV8_VERSION}.index.bc /usr/lib/postgresql/14/lib/bitcode/
COPY --from=builder /usr/lib/postgresql/14/lib/bitcode/plv8-${PLV8_VERSION}/* /usr/lib/postgresql/14/lib/bitcode/plv8-${PLV8_VERSION}/
```

Reply:
I think I'd prefer the `ENV` method.

and yeah, it's a mistake on the copy, but if you can fix it up with an `ENV` I'd be much happier :)

Reply:
Thanks for the review.
Replaced some literals with ENV.